File: app/src/main/java/seigneur/gauvain/mycourt/utils/timber/TimberLog.java
package seigneur.gauvain.mycourt.utils.timber;

import seigneur.gauvain.mycourt.BuildConfig;
import timber.log.Timber;

public class TimberLog {

    public static void init() {
        if (BuildConfig.DEBUG)
            Timber.plant(new Timber.DebugTree());
        else
            Timber.plant(new TimberReleaseTree());
    }
}

File: app/src/main/java/seigneur/gauvain/mycourt/utils/timber/TimberLog.kt
package seigneur.gauvain.mycourt.utils.timber

import seigneur.gauvain.mycourt.BuildConfig
import timber.log.Timber

object TimberLog {

    fun init() {
        if (BuildConfig.DEBUG)
            Timber.plant(Timber.DebugTree())
        else
            Timber.plant(TimberReleaseTree())
    }
}